cfme/tests/cloud_infra_common/test_tag_objects.py

```diff
@@ -147,6 +170,24 @@ def test_tag_infra_objects(tagging_check, infra_test_item, tag_place):
     tagging_check(infra_test_item, tag_place)
 
 
+@pytest.mark.meta(automates=[1726313])
+@pytest.mark.provider([InfraProvider], selector=ONE)
+def test_tag_vm_10k_category(tag_out_of_10k_values, tag_vm):
+    """ Test for https://bugzilla.redhat.com/show_bug.cgi?id=1726313
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: Tagging
+        initialEstimate: 1/6h
+
+    Bugzilla:
```

Comment:
**Required:**
`Bugzilla` tag should be above `Polarion` as It will not be populated.

Reply:
Updated, although I don't think it's documented anywhere

Reply:
I have seen most the PR's in such ways, also I have checked it in Polarion. You can also check you automated PR's in the Polarion and see the comparison. :smile: